cmd/kubeadm/app/cmd/init.go

```diff
@@ -248,6 +251,14 @@ func AddInitOtherFlags(
 		uploadCerts, options.UploadCerts, *uploadCerts,
 		"Upload certfificates to kubeadm-certs secret.",
 	)
+	flagSet.StringVar(
+		certificateKey, options.CertificateKey, "",
+		"Use this key to encrypt the certificate secrets.",
```

Comment:
`Key used to encrypt the control-plane certificates in the kubeadm-certs Secret.`